internal/fdo.go

```diff
@@ -174,6 +174,18 @@ func fdoLookupApplicationsMatching(appName string) []desktop.AppData {
 	return icons
 }
 
+func fdoLookupApplicationAll() []desktop.AppData {
```

Comment:
This uses the singlar matching fdoLookupApplicationWinInfo rather than the plural in fdoLookupApplicationsMatching which implies it's grouped with application info rather than groups of applications. Perhaps "fdoLookupApplications" is better?